### ISSUE TITLE
import matplotlib fails during runtime, can't find rcParams.

### ISSUE THREAD

**Issue by math-artist** [Nuitka/Nuitka#577]
I found a closed issue with the same behavior, but in that case matplotlib was not necessary.

I have installed matplolib 3.1.1, or to be more precise matplotlib-core, because I only need that part. So far my only usage is to get the color scales to work with pyvista/VTK.

I am using the niutka-hints.py to create stand-alones.

This code:
```
try:
    import matplotlib
    has_matplotlib = True
    print("Matplotlib is detected")
except ImportError as e:
    print("Matplotlib is NOT detected")
    print(e)
    has_matplotlib = False
```
Works fine in CPython, but I get this instead:
```
Matplotlib is NOT detected
No module named 'kiwisolver'
```

I tracked it down to this code in matplotlib init:
```
    for modname, minver in [
            ("cycler", "0.10"),
            ("dateutil", "2.1"),
            ("kiwisolver", "1.0.1"),
            ("numpy", "1.11"),
            ("pyparsing", "2.0.1"),
    ]:
        module = importlib.import_module(modname)
        if LooseVersion(module.__version__) < minver:
            raise ImportError("Matplotlib requires {}>={}; you have {}"
                              .format(modname, minver, module.__version__))
```
which may look like a new implicit import. Any thoughts?

**Comment by JorjMcKie** [comment]
please specify the relevant Nuitka context: standalone?

**Comment by JorjMcKie** [comment]
This bug should have been addressed in the latest Nuitka develop branch 0.6.7rc2
